4.4/4.4.6/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.6
+Создать объект со следующей логикой:
+Появляется сообщение: "Введите любой текст (он будет накапливаться)"
+Потом появляется конферм: "Накопленный текст: "[... ... ...]". Хотите еще добавить?"
+Когда пользователь нажмет ОК - начинается всё сначала, появляется первое сообщение.
+Если нажимает ОТМЕНА - всё заканчивается и выводится просто алерт: "Накопленный текст: "[... ... ...]""
+Каждый новый добавленнный текст отделяется от предыдущего с помощью пробела.
+<script>
+	const userTextObj = {
+		text: '',
+		isContinue : true,
+		accumulateInfo() {
+			this.text += (this.text) ? ' ' : this.text;
```

Comment:
How you can simplify this code?
`this.text += (this.text) ? ' ' : this.text;`

you can call me if it's not clear for you